prow/plugins.yaml

```diff
@@ -51,6 +51,7 @@ plugins:
   kubernetes/kubernetes:
   - trigger
   - release-note
+  - docs-no-retest
```

Comment:
@cjwagner I am sure you already know that this will require you to redeploy instantly.

Reply:
Yep, I'll merge and redeploy Prow and the kubernetes misc-mungers deployment now.
/approve